diff -ruN blatSrc/inc/common.mk blatSrc.new/inc/common.mk
--- blatSrc/inc/common.mk	2022-01-08 11:12:10.000000000 -0500
+++ blatSrc.new/inc/common.mk	2024-05-23 17:18:52.172206729 -0400
@@ -143,28 +143,8 @@
    L+=/hive/groups/browser/freetype/freetype-2.10.0/objs/.libs/libfreetype.a -lbz2
    L+=/usr/lib64/libssl.a /usr/lib64/libcrypto.a -lkrb5 -lk5crypto -ldl
 else
-   ifneq ($(wildcard /opt/local/lib/libssl.a),)
-       L+=/opt/local/lib/libssl.a
-   else
-     ifneq ($(wildcard /usr/lib/x86_64-linux-gnu/libssl.a),)
-	L+=/usr/lib/x86_64-linux-gnu/libssl.a
-     else
-        ifneq ($(wildcard /usr/local/opt/openssl/lib/libssl.a),)
-           L+=/usr/local/opt/openssl/lib/libssl.a
-        else
-           L+=-lssl
-        endif
-     endif
-   endif
-   ifneq ($(wildcard /opt/local/lib/libcrypto.a),)
-       L+=/opt/local/lib/libcrypto.a
-   else
-        ifneq ($(wildcard /usr/local/opt/openssl/lib/libcrypto.a),)
-           L+=/usr/local/opt/openssl/lib/libcrypto.a
-        else
-           L+=-lcrypto
-        endif
-   endif
+   L+=-lssl
+   L+=-lcrypto
 endif
 
 # autodetect where libm is installed
