--- arch/makefile.include.linux_nv_acc.orig	2025-10-20 17:04:17.813536310 -0400
+++ arch/makefile.include.linux_nv_acc	2025-10-20 17:07:06.076598513 -0400
@@ -75,3 +75,8 @@
 # Normally no need to change this
 SRCDIR     = ../../src
 BINDIR     = ../../bin
+
+CPP_OPTIONS+= -DVASP_HDF5
+HDF5_ROOT  ?= $(EBROOTHDF5)
+LLIBS      += -L$(HDF5_ROOT)/lib -lhdf5_fortran
+INCS       += -I$(HDF5_ROOT)/include
\ No newline at end of file
